packages/paste-core/components/form/__tests__/formlabel.test.tsx

```diff
@@ -50,8 +50,8 @@ describe('FormLabel disabled prop', () => {
 
   const container = shallow(<FormLabel {...initialProps} />);
 
-  it('should have colorTextWeak', () => {
-    expect(container.find('Text').prop('color')).toEqual('colorTextWeak');
+  it('should have colorTextWeaker', () => {
```

Comment:
I'm not sure how valuable this test is, tbh. Maybe worth removing?

Reply:
Yeah. Agreed. Probably not worth keeping in there. VRT would catch it if its broken.